heft-plugins/heft-webpack5-plugin/src/Webpack5Plugin.ts

```diff
@@ -105,64 +99,36 @@ export default class Webpack5Plugin implements IHeftTaskPlugin<IWebpackPluginOpt
     options: IWebpackPluginOptions,
     requestRun?: () => void
   ): Promise<IWebpackConfiguration | undefined> {
-    if (this._webpackConfiguration === UNINITIALIZED) {
-      // Obtain the webpack configuration by calling into the hook. If undefined
-      // is returned, load the default Webpack configuration.
-      taskSession.logger.terminal.writeVerboseLine(
-        'Attempting to load Webpack configuration via external plugins'
-      );
-      let webpackConfiguration: IWebpackConfiguration | false | undefined =
-        await this.accessor.hooks.onLoadConfiguration.promise();
-      if (webpackConfiguration === undefined) {
-        taskSession.logger.terminal.writeVerboseLine('Attempt to load the default Webpack configuration');
-        const configurationLoader: WebpackConfigurationLoader = new WebpackConfigurationLoader(
-          taskSession.logger,
-          taskSession.parameters.production,
-          taskSession.parameters.watch && this._isServeMode
+    if (this._webpackConfiguration === false) {
+      const webpackConfiguration: IWebpackConfiguration | false | undefined =
```

Comment:
Hmm... if `webpackConfiguration` can be false here, and then it's assigned to the instance variable below, doesn't that mean that the `tryLoadWebpackConfigurationAsync` method could be run multiple times, causing multiple taps of the hook?

Reply:
If the return value is `false` it gets mutated to `undefined`.

Reply:
Oops, wasn't looking at context. The function can't return `false`. Fixed.